index.html

```diff
@@ -52,7 +52,7 @@ <h2>Contact Information</h2>
     <span>Copyright 2022</span>
   </footer>
 
-  <script  src="./index.js" type="module"></script>
+  <script  src="./index.mjs" type="module"></script>
```

Comment:
- [ ] You are supposed to have an `index.js` file in your project and not an `index.mjs` file. Kindly address this issue 💯

Reply:
Yes, I am supposed to use index.js. I have been having issues using index.js. I kept getting a node_module-related error. I contacted discourse, as usual, They recommend using "index.cjs",  npx hint update to version 7 etc but none of these recommendations worked. I had to Google for a way around this issue and I got this.  It is a much more recent approach preferred by many in the community sir instead of "index.cjs" as recommended which did not work.